lib/crawl.js

```diff
@@ -80,6 +74,24 @@ function crawl(url, flags) {
         crawler.scanSubdomains = true;
     }
 
+    if (flags.template) {
+        confPath = flags.template;
+    }
+
+    const defaultConf = fs.existsSync(confPath)
+        ? JSON.parse(fs.readFileSync(confPath))
+        : require('./backstop.template.json');
+
+    /**
+     * Stash the default scenario from the template then delete it
+     * as defaultScenario isn't a valid key for backstop.json
+     */
+    const defaultScenario = defaultConf.defaultScenario;
+    delete defaultConf.defaultScenario;
+
+    // Grab any scenarios defined in the template.
+    overriddenScenarios = defaultConf.scenarios;
```

Comment:
Just to make sure, `scenarios` is part of the default template right?

Reply:
Correct. It's an empty array by default.